decode_data.py

```diff
@@ -253,3 +251,18 @@ def decode35(data: List[int]) -> Dict[int, Any]:
         103: pd.bigEndian(data[2:4]),
         104: data[4]
     }
+
+def decodeMPUStates(data: List[int]) -> Dict[int, Any]:
```

Comment:
Probably should be names more generally to MPU Dashboard Info or something, since this message will be expanded as we need more stuff from MPU sent to dash